config/production.exs

```diff
@@ -1,15 +1,68 @@
 use Mix.Config
 
- config :app_prototype, AppPrototype.Endpoint,
-   http: [port: {:system, "PORT"}],
-   url: [scheme: "https", host: "app-prototype-prod.herokuapp.com", port: 443],
-   force_ssl: [rewrite_on: [:x_forwarded_proto]],
-   cache_static_manifest: "priv/static/manifest.json",
-   secret_key_base: System.get_env("SECRET_KEY_BASE")
-
- # Configure your database
- config :app_prototype, AppPrototype.Repo,
-   adapter: Ecto.Adapters.Postgres,
-   url: System.get_env("DATABASE_URL"),
-   pool_size: String.to_integer(System.get_env("POOL_SIZE") || "10"),
-   ssl: true
+# For production, we often load configuration from external
+# sources, such as your system environment. For this reason,
+# you won't find the :http configuration below, but set inside
+# AppPrototypeWeb.Endpoint.init/2 when load_from_system_env is
+# true. Any dynamic configuration should be done there.
+#
+# Don't forget to configure the url host to something meaningful,
+# Phoenix uses this information when generating URLs.
+#
+# Finally, we also include the path to a cache manifest
+# containing the digested version of static files. This
+# manifest is generated by the mix phx.digest task
+# which you typically run after static files are built.
+
+config :app_prototype, AppPrototypeWeb.Endpoint,
+  load_from_system_env: true
+  url: [scheme: "https", host: "app-prototype-prod.herokuapp.com", port: 443],
+  force_ssl: [rewrite_on: [:x_forwarded_proto]],
+  cache_static_manifest: "priv/static/cache_manifest.json",
+  secret_key_base: System.get_env("SECRET_KEY_BASE")
+
+# ## SSL Support
```

Comment:
I threw this comment block in just because the generator provides it - did we omit it previously on purpose @craiglyons?